### ISSUE TITLE
importing ABCs from collections instead of collections.abc is deprecated

### ISSUE THREAD

**Issue by bentiss** [libratbag/ratbag-emu#29]
As seen from `pytest`:
```
========================================================= warnings summary ==========================================================
/usr/lib/python3.7/site-packages/pyudev/device/_device.py:35
/usr/local/lib/python3.7/site-packages/pyudev/device/_device.py:35: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
     from collections import Container
```

**Comment by bentiss** [comment]
Well, that seems to be a pyudev issue.

The weird part is that I do not have the same warning with hid-tools, when it should also be using pyudev...

**Comment by bentiss** [comment]
reason is: we are explicitly requesting pyudev 0.21 when the fix is in pyudev 0.22
